tbx/project_styleguide/templates/patterns/navigation/components/breadcrumbs.html

```diff
@@ -7,13 +7,18 @@
                 {% for ancestor_page in page.get_ancestors %}
                     {% if not ancestor_page.is_root %}
                         <li class="breadcrumbs-nav__item {% if ancestor_page == parent_page or ancestor_page.pattern_library_is_parent %}breadcrumbs-nav__item--parent{% else %}breadcrumbs-nav__item--ancestor{% endif %}">
-                            <a href="{% pageurl ancestor_page %}" class="breadcrumbs-nav__link">{{ ancestor_page.title }}</a>
-                            <span aria-hidden="true" class="breadcrumbs-nav__divider">/</span>
+                            {% if ancestor_page.depth > 2 %}
```

Comment:
This is making for some odd changes in the display of the breadcrumb in the pattern library, for example on a page template.

Before these changes:
![Screenshot 2024-03-12 at 11 47 31](https://github.com/torchbox/torchbox.com/assets/771869/31274b5f-9a4b-4a85-b33d-877cfa3097a3)

After:
![Screenshot 2024-03-12 at 11 47 37](https://github.com/torchbox/torchbox.com/assets/771869/b0bb92bc-262c-4403-96be-5df555f507bf)

Reply:
Updating the breadcrumb yaml to this seems to fix it:

```
context:
  page:
    title: Current page title
    get_ancestors:
      - is_root: false
        title: Some category
      - is_root: false
        depth: 3
        title: Some sub category
      - is_root: false
        depth: 4
        title: We need to go deeper
        pattern_library_is_parent: True # allows us to set a parent page just in the pattern library
```